verification/src/block_verifier.rs

```diff
@@ -97,6 +96,16 @@ impl<CP: ChainProvider + Clone> CellbaseVerifier<CP> {
             return Err(Error::Cellbase(CellbaseError::InvalidOutput));
         }
 
+        let invalid_capacity = cellbase_transaction.outputs().iter().any(|op| {
```

Comment:
I think that it's better to add a function to `CellOutput`
```
pub fn is_occupied_capacity_overflow(&self) {
    
}
```

And refactoring this code block to:
```
    if cellbase_transaction.outputs().iter().any(CellOutput::is_occupied_capacity_overflow) {
        return Err(Error::CapacityOverflow);
    }
```

Similar code of `CapacityVerifier` can be refactored to use this function also.

Reply:
Thanks. Updated and rebased.